### PR TITLE
[explorer/nodewatch] task: fix nodewatch layout issue

### PR DESCRIPTION
**Pull request by AnthonyLaw**
### What was the issue?
- nodewatch should belong to `explorer` folder

### What's the fix?
- move `nodewatch` to `explorer`
- update readme and build config path

### CODE REVIEW

**Review by Jaguar0625 — CHANGES_REQUESTED** [review]
please rebase on top of dev

File: .github/buildConfiguration.yaml [hunk 1]
```diff
@@ -7,3 +7,6 @@ builds:
 
   - name: Vanity Address Generator
     path: tools/vanity
+
+  - name: Explorer Nodewatch
+    path: explorer/nodewatch
```
Comment (Jaguar0625) [inline .github/buildConfiguration.yaml:12]: you have duplicate one above

File: .gitlintmodules [hunk 2]
```diff
@@ -1 +1 @@
-optin/puller|tools/vanity|monorepo|nodewatch
+optin/puller|tools/vanity|monorepo|explorer/nodewatch
```
Comment (Jaguar0625) [inline .gitlintmodules:1]: @0x6861746366574 do you want `[nodewatch]` or `[explorer/nodewatch]` tag?

**Review by Jaguar0625 — APPROVED** [review]

**Review by Jaguar0625 — APPROVED** [review]
👍

**Comment by AnthonyLaw** [comment]
@Jaguar0625 I just rebase onto dev

**Review by Jaguar0625 — APPROVED** [review]

**Pull request merged**